src/metrics/list_repos.rs

```diff
@@ -32,26 +34,116 @@ impl ListReposForOrg {
     }
 }
 
+impl ListReposForOrg {
+    fn to_repo(&self, repo_name: &str) -> Repo {
+        Repo {
+            graphql: self.graphql.clone(),
+            org_name: self.org_name.clone(),
+            repo_name: repo_name.to_string(),
+            start_date: self.start_date.clone(),
+            end_date: self.end_date.clone(),
+        }
+    }
+}
+
 #[async_trait]
 impl Producer for ListReposForOrg {
     fn column_names(&self) -> Vec<String> {
-        vec![String::from("Repository Name"), String::from("# of PRs")]
+        vec![
+            String::from("Organization"),
+            String::from("Repository"),
+            String::from("PRs Opened"),
+            String::from("Issues Opened"),
+            String::from("Issues Closed"),
+            String::from("Start Date"),
+            String::from("End Date"),
+        ]
     }
 
     async fn producer_task(mut self, tx: Sender<Vec<String>>) -> Result<(), eyre::Error> {
-        for repo in &self.repo_names {
-            let count_prs = util::count_pull_requests(
-                &mut self.graphql,
-                &self.org_name,
-                &repo,
-                &self.start_date,
-                &self.end_date,
-            )
+        for repo_name in &self.repo_names {
+            let mut repo = self.to_repo(repo_name);
+            let count_prs = repo.count_pulls().await?;
+            let count_issues = repo.spawn_count_issue_closures().await?;
+
+            tx.send(vec![
+                self.org_name.clone(),
+                repo_name.to_owned(),
+                count_prs.to_string(),
+                count_issues.opened.to_string(),
+                count_issues.closed.to_string(),
+                self.start_date.to_string(),
+                self.end_date.to_string(),
+            ])
             .await?;
-            tx.send(vec![repo.to_owned(), count_prs.to_string()])
-                .await?;
         }
 
         Ok(())
     }
 }
+
+#[derive(Default, Debug)]
+struct IssueClosuresCount {
+    opened: usize,
+    closed: usize,
+}
+
+#[derive(Clone, Debug)]
+struct Repo {
+    graphql: Graphql,
+    org_name: String,
+    repo_name: String,
+    start_date: Datetime,
+    end_date: Datetime,
+}
+
+impl Repo {
+    #[throws]
+    async fn spawn_count_issue_closures(&self) -> IssueClosuresCount {
+        let mut repo = self.clone();
+        let mut clone = self.clone();
+
+        let futures = vec![repo.count_issues("created"), clone.count_issues("closed")];
+
+        let resolved = futures::future::try_join_all(futures).await?;
+
+        let result = IssueClosuresCount {
+            opened: resolved[0],
+            closed: resolved[1],
+        };
```

Comment:
@rylev [offered nit](https://github-metrics.zulipchat.com/#narrow/stream/291433-general/topic/issue.20closures.20--.20PR.20.2366/near/247940202):
>  Nit: you can just deconstruct opened and closed in the line above (without needing to first bind to resolved)

Though I had trouble achieving this due to the fact that `resolved` is of type `Vec<usize>` and the compiler refused to deconstruct into a tuple.

```rust
// resolved is type `Vec<usize>`
let resolved = futures::future::try_join_all(futures).await?;
// compiler expected struct `Vec<usize>`, found tuple `(_, _)`
let (opened, closed) = resolved;
```

Would love to include a fix to this nit, if anyone has suggestions.